pmd-core/src/main/java/net/sourceforge/pmd/renderers/TextColorRenderer.java

```diff
@@ -55,8 +56,12 @@
     public static final String NAME = "textcolor";
 
     // What? TODO 7.0.0 Use a boolean property
-    public static final StringProperty COLOR = new StringProperty("color",
-            "Enables colors with anything other than 'false' or '0'.", "yes", 0);
+    // TODO should the "textcolor" renderer really support "optional" colors?
```

Comment:
On first glance, I'd say, we should remove this property, also the system property. I've added a task to #2524